Source/WTF/wtf/PlatformHave.h

```diff
@@ -479,7 +479,7 @@
 #define HAVE_MDNS_FAST_REGISTRATION 1
 #endif
 
-#if PLATFORM(COCOA) && !PLATFORM(WATCHOS) && !PLATFORM(APPLETV)
+#if PLATFORM(COCOA)
```

Comment:
There's still code that checks this? In platform independent places?

Reply:
Not after this patch. You're right, I should remove it entirely.